build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -37,112 +37,122 @@
 #
 # It can be run on a workstation but recommend using a git worktree dedicated
 # to packaging to avoid stomping on development artifacts.
-set -eu -o errtrace
+set -xeu -o errtrace
 
 this_dir="$(cd $(dirname $0) && pwd)"
 script_name="$(basename $0)"
-repo_root="$(cd $this_dir/../../ && pwd)"
-script_name="$(basename $0)"
+repo_root="$(cd "${this_dir}" && git rev-parse --show-toplevel)"
 manylinux_docker_image="${manylinux_docker_image:-gcr.io/iree-oss/manylinux2014_x86_64-release@sha256:b09c10868f846308bad2eab253a77d0a3f097816c40342bc289d8e62509bc5f9}"
 python_versions="${override_python_versions:-cp37-cp37m cp38-cp38 cp39-cp39 cp310-cp310}"
 output_dir="${output_dir:-${this_dir}/wheelhouse}"
 packages="${packages:-iree-runtime iree-runtime-instrumented iree-compiler}"
+package_suffix="${package_suffix:-}"
 
 function run_on_host() {
   echo "Running on host"
   echo "Launching docker image ${manylinux_docker_image}"
 
   # Canonicalize paths.
-  mkdir -p "$output_dir"
-  output_dir="$(cd $output_dir && pwd)"
+  mkdir -p "${output_dir}"
+  output_dir="$(cd "${output_dir}" && pwd)"
   echo "Outputting to ${output_dir}"
   mkdir -p "${output_dir}"
   docker run --rm \
-    -v "${repo_root}:/main_checkout/iree" \
-    -v "${output_dir}:/wheelhouse" \
+    -v "${repo_root}:${repo_root}" \
+    -v "${output_dir}:${output_dir}" \
```

Comment:
Eventually we may want to set UID/GID so we don't leave root:root owned files around if we decide to run this on a workstation to test. I added some flags here: https://github.com/llvm/torch-mlir/pull/1234

Reply:
Yeah we do that for all our other docker stuff. This was developed totally separately and does a lot of things differently